docs/installation/troubleshooting/bootstrap_installation_errors.rst

```diff
@@ -34,8 +36,134 @@ salt-master container) do not start, perform the following checks:
+
+You can solve this issue by either adding a default route to your host or
+adding a dummy network interface used to define a route for this network.
+
+Configuring a default route:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To configure a default route, please, refer to the official documentation of
+your Linux distribution.
+
+Configuring a dummy interface:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**CentOS / RHEL 7**
+
+Create the ``dummy-metalk8s`` interface configuration:
+
+  .. code-block:: shell
+
+     cat > /etc/sysconfig/network-scripts/ifcfg-dummy-metalk8s << 'EOF'
+     ONBOOT=yes
+     DEVICE=dummy
+     NM_CONTROLLED=no
+     NAME=dummy-metalk8s
+     EOF
+
+Create the ``ifup-dummy`` network script:
+
+  .. code-block:: shell
+
+     cat > /etc/sysconfig/network-scripts/ifup-dummy << 'EOF'
+     #!/bin/sh
+     # Network configuration file for dummy network interface
+
+     . /etc/init.d/functions
+
+     cd /etc/sysconfig/network-scripts
+     . ./network-functions
+
+     [ -f ../network ] && . ../network
+
+     CONFIG=${1}
+
+     need_config "${CONFIG}"
+     source_config
+
+     modprobe --first-time ${DEVICETYPE} numdummies=0 2> /dev/null || echo dummy module already loaded
+     ip link add ${DEVNAME} type ${DEVICETYPE}
+     [[ -n "${IPADDR}" && -n "${NETMASK}" ]] && ip address add ${IPADDR}/${NETMASK} dev ${DEVNAME}
+     ip link set ${DEVNAME} up
+     /etc/sysconfig/network-scripts/ifup-routes ${DEVICE} ${NAME}
+     EOF
+
+     chmod +x /etc/sysconfig/network-scripts/ifup-dummy
+
+Create the ``ifdown-dummy`` network script:
+
+  .. code-block:: shell
+
+     cat > /etc/sysconfig/network-scripts/ifdown-dummy << 'EOF'
+     #!/bin/sh
+     . /etc/init.d/functions
+
+     cd /etc/sysconfig/network-scripts
+     . ./network-functions
+
+     [ -f ../network ] && . ../network
+
+     CONFIG=${1}
+
+     need_config "${CONFIG}"
+
+     source_config
+
+     ip link set ${DEVNAME} down
+     ip link del ${DEVNAME} type ${DEVICETYPE}
+     EOF
+
+     chmod +x /etc/sysconfig/network-scripts/ifdown-dummy
+
+Create the ``route-dummy-metalk8s`` network script:
+
+  .. code-block:: shell
+
+     cat > /etc/sysconfig/network-scripts/route-dummy-metalk8s << EOF
+     $(salt-call pillar.get networks:service --out=txt | cut -d' ' -f2-) dev dummy-metalk8s
+     EOF
+
+Start the ``dummy-metalk8s`` interface:
+
+  .. code-block:: shell
+
+     ifup dummy-metalk8s
+
+**CentOS / RHEL 8 (and other NetworkManager based dists)**
+
+Retrieve the service network CIDR:
+
+  .. code-block:: shell
+
+     salt-call pillar.get networks:service --out=txt | cut -d' ' -f2-
+
+Create the ``dummy-metalk8s`` interface:
+
+  .. code-block:: shell
+
+     nmcli connection add type dummy ifname dummy-metalk8s ipv4.method manual ipv4.addresses <dummy-iface-ip> ipv4.routes <network-cidr>
+
+  .. note::
+
+     Replace ``<dummy-iface-ip>`` by any available IP in the previously
+     retrieved network CIDR (e.g. 10.96.0.1 for a 10.96.0.0/12 network CIDR)
```

Comment:
not 10.96.0.1, as it looks like it's conflicting with `kubernetes.default.svc.cluster.local`

Reply:
done